packages/dht/src/dht/discovery/DiscoverySession.ts

```diff
@@ -97,16 +83,18 @@ export class DiscoverySession {
             return
         }
         this.ongoingClosestPeersRequests.delete(peer.getPeerId().toKey())
-        this.config.bucket.remove(peer.getPeerId().value)
-        this.config.neighborList.removeContact(peer.getPeerId())
+
+        this.config.peerManager.handlePeerUnresponsive(peer)
     }
 
     private findMoreContacts(): void {
         if (this.stopped) {
             return
         }
-        const uncontacted = this.config.neighborList.getUncontactedContacts(this.config.parallelism)
+        const uncontacted = this.config.peerManager.getClosestPeersTo(this.config.targetId, this.config.parallelism, this.contactedPeers)
+
         if (uncontacted.length < 1 || this.noProgressCounter >= this.config.noProgressLimit) {
+            logger.trace( this.config.nodeName + ' discoveryCompleted in findMoreContacts, path: ' + this.joiningPath.join(', '))
```

Comment:
Use node id instead of `nodeName`, see https://github.com/streamr-dev/network/pull/1967

(also at line 133, and remove the field from `DiscoverySessionConfig`)

Reply:
done